### DISCUSSION TITLE
Launch single batch job for multiple parameter combinations

### DISCUSSION THREAD

**Discussion by dsfenn** [llnl/maestrowf#294] [Q&A]
I only recently started using Maestro, so it could be that I'm using it incorrectly. I'm trying to run a parameter space study that involves several thousand relatively cheap runs, where each individual run is serial. I followed the LULESH example for the parallel setup. What's happening is that Maestro is launching a new job for every parameter combination. This ends up being wasteful and slow, since each run is serial, so most of the processors on the node go unused.

Is there a way to launch a singe job on a specified number of nodes that will process all the parameter combinations?

Thanks

**Comment by FrankD412** [comment]
Hi @dsfenn! -- Right now Maestro supports running parameter studies. It sounds like you'd like the conductor to be launched as a kind of worker on a set of nodes. That functionality is currently in an alpha phase, and is implemented by @jwhite242. We can point you to it if you want to try it out, but this mode of operating is not in production yet.

**Comment by dsfenn** [comment]
@FrankD412 -- that would be great! I'm interested in trying it out, even if it's only in alpha. How can I get started with that?
